PerfNext/app.js

```diff
@@ -9,11 +9,11 @@ global.appRoot = path.resolve(__dirname);
 global.APP_DATA = JSON.parse(fs.readFileSync('config/APP_DATA.json', 'utf8'));
 
 /* Refer to PerfNext/README.md for Password Requirements */
-var pwdList = fs.readFileSync('perfnext_passwords', 'utf8');
-global.perffarmEspressoPwd = pwdList.split(' ')[0].substring(0, 8);
-global.jenkinsPwd = pwdList.split(' ')[1].substring(0, 8);
-global.perffarmUsername = '';
-global.jenkinsUsername = '';
+global.credentials = JSON.parse(fs.readFileSync('config/credentials.json', 'utf8'));
```

Comment:
Seems like these changes got missed when we open-sourced the code from our internal repo. I'll find the mssing changes and merge them later.